src/pages/workspaces/workspace/analysis/modals/AzureComputeModal.js

```diff
@@ -227,6 +228,16 @@ export const AzureComputeModalBase = ({
           labels: { saturnWorkspaceNamespace: namespace, saturnWorkspaceName: workspaceName }
         }
 
+        Ajax().Metrics.captureEvent(Events.analysisAzureJupyterLabCreate, {
+          region: computeConfig.region,
+          machineSize: computeConfig.machineType,
+          labels: {
+            saturnWorkspaceNamespace: namespace,
+            saturnWorkspaceName: workspaceName
+          },
+          disk
```

Comment:
I don't know if we want to send the full disk here, maybe just the size? Also, I'd consider adding the workspaceId.

Reply:
Made an adjustment and flattened it a bit!